src/styles/index.scss

```diff
@@ -1,3 +1,109 @@
+@import url(./stars.scss);
```

Comment:
create main.scss where will be all imports

Reply:
Not fixed, read the checklist 11